### DISCUSSION TITLE
Does you galaxy brain stay forever, or will it go away if your answer is "unmarked"?

### DISCUSSION THREAD

**Discussion by jherskow** [Schweinepriester/github-profile-achievements#226] [Q&A]
If you try and answer, I will Endeavour to test it out.

**Comment by mezotv** [comment]
I think it does given that the x discussion answered also stays if you remove it

**Reply by jherskow** [reply]
marked and unmarked, lets see

**Comment by thenithinbalaji** [comment]
If the entire discussion is deleted -> the count for badge will stay
If the answer is unmarked, -> the count is reduced

**Reply by thenithinbalaji** [reply]
@jherskow

**Comment by melvincwng** [comment]
@jherskow, it remains in your profile page even if the answer is unmarked.

Hope this clarifies.
